src/components/Search/SearchPageHeader.tsx

```diff
@@ -314,7 +314,14 @@ function SearchPageHeader({queryJSON, hash, onSelectDeleteOption, setOfflineModa
             <Button
                 text={translate('search.filtersHeader')}
                 icon={Expensicons.Filters}
-                onPress={() => Navigation.navigate(ROUTES.SEARCH_ADVANCED_FILTERS)}
+                onPress={() => {
+                    const filters = SearchUtils.getFilters(queryJSON);
+                    const form = SearchUtils.getFiltersForm(filters);
```

Comment:
```suggestion
                    const values = SearchUtils.getFiltersForm(filters);
```

Totally not a bug, but I checked in the code and we almost always call `updateAdvancedFilters` with the argument `values` so I thought maybe we should always refer to this object as values 🤷